src/App.js

```diff
@@ -1,24 +1,38 @@
-import React, {useEffect, useState} from "react";
+import React, {useEffect, useRef, useState} from "react";
 import FlashcardList from "./FlashcardList";
 import FlashcardData from "./data/flashcards.json";
+import LanguageData from "./data/languages.json"
+//import axios from "request";
 //import 'bootstrap/dist/css/bootstrap.min.css';
 
+
 function App() {
-  const [flashcards] = useState(FlashcardData)
-  const [languages, setLanguages] = useState(FlashcardData)
+  const [flashcards,setFlashcards] = useState(FlashcardData)
+  const [languages, setLanguages] = useState(LanguageData)
+
+    const languageEl = useRef(LanguageData)
+    const amountEl = useRef(FlashcardData)
+
+    const CardShuffler = (arr) => {
+        for (let i = arr.length - 1; i > 0; i--) {
+            const j = Math.floor(Math.random() * (i + 1));
+            [arr[i], arr[j]] = [arr[j], arr[i]];
+        }
+        return arr;
+    };
 
   useEffect(() =>{
 
-        setLanguages(FlashcardData)
+        setLanguages(LanguageData)
 
   }, [])
 
   return (
       <>
-      <form className="header">
+      <form className="header" /*onSubmit={handleSubmit}*/>
         <div className="form-group">
           <label htmlFor="language">Language that you know</label>
-          <select id="language">
+          <select id="language" ref={languageEl}>
             {languages.map(language => {
               return <option value={language.id} key={language.id}>{language.name}</option>
             })}
```

Comment:
Anti-pattern !!!

Try to keep the html clean. Without conditions and loops.  This should be: 
```jsx 
// before return 
const languageOptions = languages.map(language => {
              return <option value={language.id} key={language.id}>{language.name}</option>
            }); 
//...
return (
...
{languageOptions}
...
);

```